packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -104,6 +104,11 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
                         this.editor.insertNode(htmlNode);
                     }
                     this.setIsSuggesting(false);
+
+                    return {
```

Comment:
How we are going to use this info?

Reply:
I am planning to use this info to announce to a screen reader the element inserted with PickerPlugin after auto-correcting a emoji.

Reply:
I think I found another way to do it. Without this change. I will close this now.